data/src/test/java/org/apache/iceberg/TestMergingMetrics.java

```diff
@@ -91,8 +87,7 @@ public abstract class TestMergingMetrics<T> {
               optional(27, "timestamp", Types.TimestampType.withZone())));
 
   private static final Map<Types.NestedField, Integer> FIELDS_WITH_NAN_COUNT_TO_ID =
-      ImmutableMap.of(
-          FLOAT_FIELD, 3, DOUBLE_FIELD, 4, FLOAT_LIST, 10, MAP_FIELD_1, 18, MAP_FIELD_2, 22);
+      ImmutableMap.of(FLOAT_FIELD, 3, DOUBLE_FIELD, 4);
```

Comment:
Like ORC, Parquet will no longer produce metrics for repeated fields, like those in map keys or values or in list elements.